jax/_src/numpy/lax_numpy.py

```diff
@@ -4067,6 +4067,26 @@ def polyint(p, m=1, k=None):
     return true_divide(concatenate((p, k)), coeff)
 
 
+@_wraps(np.polydiv)
+def polydiv(u,v):
+  u = asarray(u) + 0.0
+  v = asarray(v) + 0.0
+
+  w = u[0] + v[0]
+  m = len(u) - 1
+  n = len(v) - 1
+  scale = 1. / v[0]
+  q = zeros((max(m - n + 1, 1),), w.dtype)
+  r = u.astype(w.dtype)
+  for k in range(0, m-n+1):
+    d = scale * r[k]
+    q[k] = d
+    r[k:k+n+1] -= d*v
+  while allclose(r[0], 0, rtol=1e-14) and (r.shape[-1] > 1):
+    r = r[1:]
```

Comment:
I am facing issues with this line 
```
q = zeros((max(m - n + 1, 1),), w.dtype)
```
It is showing me a Value error 

also what other way would you suggest for assignment in JAX with reference to  r[k:k+n+1] -= d*v

Reply:
`jax.numpy` overrides `max` to match the behavior of `numpy.max`. If you want the builtin `max` function, use `_max` here.

`r[k:k+n+1] -= d * v` will not work because JAX arrays are immutable. You'll need to use [index update])https://jax.readthedocs.io/en/latest/jax.ops.html#indexed-update-operators) operations instead; e.g.
```python
r = r.at[k:k+n+1].add(-d*v)
```